src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
```

Comment:
Yes, `_primeMover` is now the `WorkflowManager`.  `_primeMover` has been on develop for a while I believe, so it wouldn't be new to these diffs.